recipes/blas/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set blas_pkg = "openblas" %}
+
+{% set version = "1.0.0" %}
+{% set build_number = 0 %}
+
+{% set version_post = (".post" + build_number|string) if build_number|int != 0 else "" %}
+{% set version_full = version + version_post %}
```

Comment:
I really would encourage you not to strive so hard to treat these like normal packages. They're definitely not, and it would be just fine to define a separate standard for them. In fact, if this works well, we may bury some of these details into conda syntactic sugar.